src/components/dashboard/dashboard-page-client.tsx

```diff
@@ -27,87 +30,93 @@ export function DashboardPageClient() {
     router.push('/review');
   };
 
+  const canAnnotate =
+    permissions && canViewAndWriteAnnotationTasks(permissions);
+  const canReview = permissions && canViewSites(permissions);
+  const hasAnyAccess = canAnnotate || canReview;
+
+  if (!hasAnyAccess) {
```

Comment:
If they have no access, instead of null, it might be better to return something to avoid them needing to stare at a white screen :)